features/hr.yml

```
@@ -1,6 +1,5 @@
-draft_date: 2024-06-05
 name: <hr>
-description: The `<hr>` element TODO.
+description: The `<hr>` element represents a thematic break between paragraph-level elements.
```

Two ideas here:

- to aim for brevity on the main concept here (paragraphs)
- to acknowledge the historic abbreviation (for search) and the default behavior

```suggestion
description: The `<hr>` element represents a thematic break between paragraphs, often shown as a horizontal rule.
```